man/import.q.sorts.Rd

```diff
@@ -17,69 +18,75 @@
+		A matrix with meaningful short handles (same as in \code{\link{build.q.set}}, \code{\link{import.q.concourse}}) as rownames, and arbitary, unique identifying strings (ID) in any of the columns as printed on the q-cards by \code{\link{make.cards}}.
+		Defaults to \code{NULL} in which case items are automatically identified by full item hashes, as also used by \code{\link{make.cards}}.
 	}
 }
 
 \details{
-Q-sorts in their raw form, as created by participants, have columns as ranks (from, say, -6 to +6) with cards (items) sorted in rows.
-The vertical dimension of q-sorts is \emph{meaningless}.
-For reproduceable research, q-sorts, as all data, should be maintained in their rawest form and any processing should be well-documented.
+	Q sorts in their raw form, as created by participants, have columns as ranks (from, say, \code{-6} to \code{+6}) with cards (items) sorted in rows.
+	The vertical dimension of Q sorts is \emph{meaningless}.
+	For reproduceable research, Q sorts, as all data, should be maintained in their rawest form and any processing should be well-documented.
 
-That is what this function offers, by reading in q-sorts from the form in which they were entered by participants.
+	That is what this function offers, by reading in Q sorts from the form in which they were entered by participants.
 
-Q-sorts are best entered not by typing up the full form of an item, but some unique string printed on the card.
-So as \emph{not} to influence participants, these strings should be meaningless to humans.
-This function offers either a set of arbitrary strings hard entered by a researcher for identification in \code{manual.lookup}, or expects a cryptographic hash of the complete item wording, as conveniently also created by \code{\link{make.cards}}.
-These hash values change whenever \emph{anything} in the full item wordings is changed, and allow a precise idenfication of different versions of an item.
+	Q sorts are best entered not by typing up the full form of an item, but some unique string (ID) printed on the card.
+	So as \emph{not} to influence participants, these strings should be meaningless to humans.
+	This function offers either a set of arbitrary strings hard entered by a researcher for identification in \code{manual.lookup}, or expects a cryptographic hash of the complete item wording, as conveniently also created by \code{\link{make.cards}}.
+	These hash values change whenever \emph{anything} in the full item wordings is changed, and allow a precise idenfication of different versions of an item.
 
-Q-sorts are conveniently entered as \code{*.csv} (comma separated values) in popular spreadsheet editors.
-This function will ignore any rows in the files below the maximum height of columns expected from the q-distribution.
+	Q sorts are conveniently entered as *.CSV (comma separated values) files in popular spreadsheet editors.
```

Comment:
standard/ regular spreadsheet editors ? (LibreCalc may not be popular but still a spreadsheet editor)